subcommands/devices/config_wireguard.go

```diff
@@ -117,7 +117,7 @@ func findVpnAddress(factory string) string {
 
 	ips := factoryIps(factory)
 	for ip := serverIp + 1; ip < serverIp+10000; ip++ {
-		if _, ok := ips[ip]; !ok {
+		if _, ok := ips[ip]; !ok && byte(ip) != 0 {
```

Comment:
Do we allow `.1` device IP?

Reply:
Which network mask do we use for VPN? Shouldn't we test the host part of the address, not just the last byte?

Reply:
wireguard isn't putting devices on a subnet. they each just have a unique IP and route to one other in a mesh.

I was a little surprised, but .1 worked fine. It was just a .0 that tripped things up

Reply:
Oh, it's a mesh. Thanks, it's interesting, need to read more about WG :)